src/functions/module_lookup.ts

```diff
@@ -32,6 +36,58 @@ function assertAvailable(variable, searchKey) {
   }
 }
 
+// -----------------------------------------------------------------------------
+// ADDRESS
+// -----------------------------------------------------------------------------
+
+export const ADDRESS: AddFunctionDescription = {
+  description: _lt("Returns a cell reference as a string. "),
+  args: [
+    arg("row (number)", _lt("The row number of the cell reference. ")),
+    arg(
+      "column (number)",
+      _lt("The column number (not name) of the cell reference. A is column number 1. ")
+    ),
+    arg(
+      `absolute_relative_mode (number, optional, default=${ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE})`,
+      _lt(
+        "An indicator of whether the reference is row/column absolute. 1 is row and column absolute (e.g. $A$1), 2 is row absolute and column relative (e.g. A$1), 3 is row relative and column absolute (e.g. $A1), and 4 is row and column relative (e.g. A1)."
+      )
+    ),
+    arg(
+      "use_a1_notation (boolean, optional, default=TRUE)",
+      _lt(
+        "A boolean indicating whether to use A1 style notation (TRUE) or R1C1 style notation (FALSE)."
+      )
+    ),
+    arg(
+      "sheet (string, optional)",
+      _lt("A string indicating the name of the sheet into which the address points.")
+    ),
+  ],
+  returns: ["STRING"],
+  compute: function (
+    row: PrimitiveArgValue,
+    column: PrimitiveArgValue,
+    absoluteRelativeMode: ABSOLUTE_RELATIVE_MODE = ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE,
+    useA1Notation: PrimitiveArgValue = true,
+    sheet: string | undefined
+  ): string {
+    const _useA1Notation = toBoolean(useA1Notation);
+    const rowNumber = strictToPositiveNumber(row);
+    const colNumber = strictToPositiveNumber(column);
+    let cellReference = toA1Notation(rowNumber, colNumber, absoluteRelativeMode);
```

Comment:
with ```absoluteRelativeMode``` as PrimitiveArgValue you will need to transform it into a number (use ```toNumber```) and assert it is included between 1 and 4